src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    static Ball getRandomBall() {
+        Color randomColor = ColorSupplier.getRandomColor();
+        int randomNumber = new Random().nextInt(100) + 1;
+        return new Ball(randomColor, randomNumber);
+    }
+}
+

```

Comment:
remove extra empty line at the end
add private access modifier to the method
extract 100 to class constant
random and colorSupplier should both be class fields